ingestors/http.go

```diff
@@ -8,13 +8,18 @@ import (
 	"github.com/mmcdole/gofeed"
 )
 
-func depperGetUrl(url string) (*http.Response, error) {
```

Comment:
I think I'd add `depperGetUrlWithHeaders()` and keep the signature of `depperGetUrl` simpler